tsconfig.json

```diff
@@ -22,6 +22,7 @@
     "noFallthroughCasesInSwitch": true,
     "forceConsistentCasingInFileNames": true,
     "importsNotUsedAsValues": "error",
+    "useDefineForClassFields": true,
```

Comment:
Vite enables this option when it bootstraps a new project: https://www.typescriptlang.org/tsconfig#useDefineForClassFields. Sounds like a good option to enable, so I do so in the root `tsconfig`.

Reply:
I struggle to see what kind of changes it brings (especially in our code) but all right